src/Components/Action.vue

```diff
@@ -55,16 +56,13 @@ export default {
 				this.$parent._$el.fadeOut(OC.menuSpeed)
 				this.$parent.$emit('remove')
 				try {
-					$('body').trigger(new $.Event('OCA.Notification.Action', {
-						notification: this.$parent,
-						action: {
-							url: this.link,
-							type: type,
-						},
-					}))
-				// do not do anything but log, the action went fine
-				// only the event bus listener failed, this is not our problem
+					emit('notifications:action', {
```

Comment:
action … what? where is the verb? :stuck_out_tongue_closed_eyes:

Reply:
As per new naming convention the name should be `notifications:action:executed`